contrib/check-config.sh

```diff
@@ -183,7 +183,7 @@ flags=(
 	DEVPTS_MULTIPLE_INSTANCES
 	CGROUPS CGROUP_CPUACCT CGROUP_DEVICE CGROUP_FREEZER CGROUP_SCHED CPUSETS MEMCG
 	KEYS
-	MACVLAN VETH BRIDGE BRIDGE_NETFILTER
+	VETH BRIDGE BRIDGE_NETFILTER 
```

Comment:
whoa @tianon has found a new partner in crime ;)

On Thu, Jul 7, 2016 at 8:09 AM, Justin Cormack <notifications@github.com>
wrote:

> In contrib/check-config.sh
> <https://github.com/docker/docker/pull/24301#discussion_r69924213>:
>
> > @@ -183,7 +183,7 @@ flags=(
> >  	DEVPTS_MULTIPLE_INSTANCES
> >  	CGROUPS CGROUP_CPUACCT CGROUP_DEVICE CGROUP_FREEZER CGROUP_SCHED CPUSETS MEMCG
> >  	KEYS
> > -	MACVLAN VETH BRIDGE BRIDGE_NETFILTER
> > +	VETH BRIDGE BRIDGE_NETFILTER
>
> there is a trailing space here....
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/docker/pull/24301/files/04a8805413da130382d2f95f2a612fede30ab385#r69924213>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABYNbPBQR9U9_sVyKc3_-b0jrLrWESLOks5qTRaggaJpZM4JEFn3>
> .
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>